phoenix-core/src/main/java/org/apache/phoenix/iterate/TableResultIterator.java

```diff
@@ -186,20 +188,45 @@ public Tuple next() throws SQLException {
                 if (lastTuple != null) {
                     ImmutableBytesWritable ptr = new ImmutableBytesWritable();
                     lastTuple.getKey(ptr);
+                    if (scan.getStartRow() != null
+                            && Bytes.compareTo(scan.getStartRow(), HConstants.EMPTY_START_ROW) != 0) {
+                        if (scan.includeStartRow()) {
```

Comment:
What is the difference between `scan.getStartRow() != null` and `scan.includeStartRow()` ? 
Can we exclude one check if both are the same?